his_checker.py

```diff
@@ -9,22 +9,35 @@
 
 import sys
 import argparse
+import os
 
-METRIC_FAIL = 0
+CYCLOMATIC_THRESHOLD = 10
 
-def process_metric(filenames):
+def process_complexity(filename):
 
-    """Process each file."""
-    print("Checking X metric")
-    for file in filenames.files:
-        print(file)
+    """Process McCabe Cyclomatic Complexity for each function in a file"""
+
+    metric_fail = 0
+    complexity = "pmccabe -c "
```

Comment:
I would change the `complexity` variable to something more clear e.g., `cc_tool`.

Reply:
It sure gets easier to understand the argument of the variable that way. Thanks!